Lib/test/test_multiprocessing_main_handling.py

```diff
@@ -58,7 +64,8 @@ def f(x):
     p = Pool(5)
     results = []
     p.map_async(f, [1, 2, 3], callback=results.extend)
-    deadline = time.time() + 60 # up to 60 s to report the results
+    wait_in_secs = 10 * TIMEOUT_MULTIPLIER # up to 10 s to report the results
+    deadline = time.time() + wait_in_secs
```

Comment:
Please use time.monotonic() rather than time.time() to implement a timeout to behave correctly when the system clock is updated.